### DISCUSSION TITLE
Sometimes when I want to see a component source on the homepage, I get page not found.

### DISCUSSION THREAD

**Discussion by oauch** [chakra-ui/chakra-ui#8643] [General]
### Description

For example, the source code of the <a href="https://v2.chakra-ui.com/docs/components/text">Text component</a> is.
If you press the source of the component, page not found occurs.

This is often the case among components

### Link to Reproduction

https://v2.chakra-ui.com/docs/components/text

<a href="https://github.com/chakra-ui/chakra-ui/tree/v2/packages/components/src/layout">Problem Link</a>

### Chakra UI Version

2.8.2

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
Closing. This is stale and we are not acting on it.
